src/Weeds3dViewer/Weeds3dViewer.js

```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { callAzureFunction } from '../utils/SharedFunctions';
+import { useAuth0 } from '../Auth/react-auth0-spa';
+import { Grid, Snackbar } from '@material-ui/core';
+import MaterialTable from 'material-table';
+import Typography from '@material-ui/core/Typography';
+import IssueDialogue from '../Comments/IssueDialogue';
+import { useSelector } from 'react-redux';
+import { CustomLoader } from '../utils/CustomComponents';
+import { onfarmAPI } from '../utils/api_secret';
+import MuiAlert from '@material-ui/lab/Alert';
+
+// Helper function
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const tableHeaderOptions = [
```

Comment:
move this inside the Weeds3dViewer component. global table headers cause super poor performance on material tables